packages/core/src/machines/actors/auth/resetPassword.ts

```diff
@@ -0,0 +1,115 @@
+import { createMachine, assign, sendUpdate } from 'xstate';
+
+import { AuthEvent, ResetPasswordContext } from '../../../types';
+import { Auth } from 'aws-amplify';
+
+export const resetPasswordActor = createMachine<
+  ResetPasswordContext,
+  AuthEvent
+>(
+  {
+    id: 'resetPasswordActor',
```

Comment:
That's a good point. To be very honest, I just suffixed `actor` to the ids because I preferred

`signInActor.signIn` vs `signIn.signIn` to distinguish substate names from the id.

Reply:
I agree with naming the variable name `resetPasswordMachine`. For id, let me think for a minute. Maybe `signIn.signIn` is okay... or `signInMachine.signIn`?

Reply:
Let's descope this to another PR, because it's mostly naming things. But I do agree!